challenge.py

```diff
@@ -0,0 +1,9 @@
+file_object =open("a.txt","r")
+data=file_object.read().split()
+dict = {}
```

Comment:
do not use "dict" as variable name, and as general guideline - var names need to be clear and meaningful